### ISSUE TITLE
Can't install with symfony4

### ISSUE THREAD

**Issue by liverbool** [voku/anti-xss#28]
Composer install error:

```
composer require voku/anti-xss
Using version ^3.1 for voku/anti-xss
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for __root__ dev-master -> satisfiable by __root__[dev-master].
    - voku/anti-xss 3.1.0 requires voku/portable-utf8 ~4.0 -> satisfiable by voku/portable-utf8[4.0.0, 4.0.1].
    - voku/portable-utf8 4.0.0 requires symfony/polyfill ~1.0 -> satisfiable by symfony/polyfill[v1.0.0, v1.0.1, v1.1.0, v1.1.1, v1.2.0, v1.3.0, v1.3.1, v1.4.0, v1.5.0, v1.6.0].
    - voku/portable-utf8 4.0.1 requires symfony/polyfill ~1.0 -> satisfiable by symfony/polyfill[v1.0.0, v1.0.1, v1.1.0, v1.1.1, v1.2.0, v1.3.0, v1.3.1, v1.4.0, v1.5.0, v1.6.0].
    - don't install symfony/polyfill v1.0.0|remove __root__ dev-master
    - don't install symfony/polyfill v1.0.1|remove __root__ dev-master
    - don't install symfony/polyfill v1.1.0|remove __root__ dev-master
    - don't install symfony/polyfill v1.1.1|remove __root__ dev-master
    - don't install symfony/polyfill v1.2.0|remove __root__ dev-master
    - don't install symfony/polyfill v1.3.0|remove __root__ dev-master
    - don't install symfony/polyfill v1.3.1|remove __root__ dev-master
    - don't install symfony/polyfill v1.4.0|remove __root__ dev-master
    - don't install symfony/polyfill v1.5.0|remove __root__ dev-master
    - don't install symfony/polyfill v1.6.0|remove __root__ dev-master
    - Installation request for voku/anti-xss ^3.1 -> satisfiable by voku/anti-xss[3.1.0].


Installation failed, reverting ./composer.json to its original content.
```

**Comment by voku** [comment]
Thanks for the bug-report. I have published a new version 4.0.0, which should work with symfony4.

**Comment by liverbool** [comment]
Thanks, not it work!
